package.json

```diff
@@ -25,12 +25,16 @@
     "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)"
   ],
   "dependencies": {
+    "@types/hast": "^2.0.0",
+    "@types/mdast": "^3.0.0",
+    "@types/unist": "^2.0.0",
```

Comment:
minor: types are best placed in `devDependencies`

Reply:
We used to do that (in part because I pushed for it), but we changed to now include them to improve DX of TS users (except for peers such as `unified`), especially because they’re now loaded from the core code (https://github.com/rehypejs/rehype-remark/blob/cf1b894934f1d2b3b3e5dab99bf4a834933e77bf/index.js#L102-L104).

Do you see a reason to not add them as regular deps here?

Reply:
I just thought this was still the way, as it has been in most projects I've seen. But if it's been discussed it can of course be kept.